### PR TITLE
Use intlFormatDistance instead of formatDistanceToNow

### PR DESCRIPTION
**Pull request by lubej**
| Before  | After |
| ------------- | ------------- |
| image  | image  |

### CODE REVIEW

**Comment by lubej** [comment]
image

@lukaw3d here is the snapshot diff. Even though the current implementation, is not as granular. And there is no nice way to "block" quarters.

**Review by lukaw3d — COMMENTED** [review]
:/ quarters

File: src/app/utils/dateFormatter.ts [hunk 1]
```diff
@@ -1,67 +1,21 @@
...
-  return dateFnsFormatDistanceToNow(date, {
-    ...opts,
+  return intlFormatDistance(date, baseDate, {
+    style: 'short',
+    numeric: 'always',
+    locale,
   })
+    .replace(/ago/, '')
+    .replace(/in/, '')
+    .replace(/\./, '')
```
[the remark this answers is not in the document]
Comment (lubej) [reply src/app/utils/dateFormatter.ts:49]: image
This is the reason. 

It does go away with style 'narrow', but the style only uses one letter.
image

File: src/app/utils/dateFormatter.ts [hunk 2]
```diff
@@ -1,67 +1,21 @@
...
-
-  return dateFnsFormatDistanceToNow(date, {
-    ...opts,
+  return intlFormatDistance(date, baseDate, {
+    style: 'short',
+    numeric: 'always',
+    locale,
   })
+    .replace(/ago/, '')
+    .replace(/in/, '')
```
[the remark this answers is not in the document]
Comment (lubej) [reply src/app/utils/dateFormatter.ts]: Not really. But just wanted to add it to be safe.

File: src/app/utils/dateFormatter.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukaw3d) [reply src/app/utils/dateFormatter.ts:49]: ah. I'd make that /\.$/ too

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**